documentation/docs/how_tos/accounts_and_addresses/02_generate_address.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Generate an Address
+description: 'How to generate an address for a wallet account.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- generate address
+- generate wallet address
+- account manager
+- multi account approach
+- single account approach
+- nodejs
+- python
+- rust
+---
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../libraries/nodejs/how_to/accounts_and_addresses/_02_generate_address/_code_sample.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_to/accounts_and_addresses/_02_generate_address/_expected_output.md";
+import PythonCode from "../../libraries/python/how_to/accounts_and_addresses/_02_generate_address/_code_sample.mdx";
+import PythonOutput from "../../libraries/python/how_to/accounts_and_addresses/_02_generate_address/_expected_output.md";
+import RustCode from "../../libraries/rust/how_to/accounts_and_addresses/_02_generate_address/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_to/accounts_and_addresses/_02_generate_address/_expected_output.md";
+import TipAccountApproaches from "../../_admonitions/_tip_account_approaches.md";
+import IOTAJsExample from "../../_admonitions/_info_iotajs.mdx";
+
+After you have [created an account](01_create_a_wallet_account.mdx), you are ready to generate your first address.
```

Comment:
Not sure how to phrase it best, but during account creation one address is generated, so here it's not the first address anymore
```suggestion
After you have [created an account](01_create_a_wallet_account.mdx), you have a single address in the account. But you can generate more addresses.
```